sdk/resourcemanager/azure-resourcemanager-cdn/src/main/java/com/azure/resourcemanager/cdn/implementation/CdnEndpointImpl.java

```diff
@@ -72,6 +84,15 @@ public String id() {
     @Override
     public Mono<CdnEndpoint> createResourceAsync() {
         final CdnEndpointImpl self = this;
+        if (isStandardMicrosoftSku()
+            && this.innerModel().deliveryPolicy() == null
+            && this.standardRulesEngineRuleMap.size() > 0) {
+            this.innerModel().withDeliveryPolicy(new EndpointPropertiesUpdateParametersDeliveryPolicy()
+                .withRules(this.standardRulesEngineRuleMap.values()
+                    .stream()
+                    .sorted(Comparator.comparingInt(DeliveryRule::order))
```

Comment:
Does it require sort?

Reply:
Yes, strangely.. If not sorted(2,1,3), or even not the sequential order(1,2,4), service will reject the request.

I thought of arrange the order by ourselves, but it introduces similar complexities like in previous ApplicationGatewayRequestRoutingRule..